reference/carts.v3.yml

```diff
@@ -1791,7 +1787,11 @@ components:
               message:
                 type: string
                 description: Limited to 200 characters.
```

Comment:
Need to add more detail for the description: "The message included in the gift certificate limited to 200 characters."